release_files/ui-post-install.sh

```
@@ -1,10 +1,12 @@
 #!/bin/sh
 
 # Check if netbird-ui is running
-if pgrep -x -f /usr/bin/netbird-ui >/dev/null 2>&1;
+pid="$(pgrep -x -f /usr/bin/netbird-ui)"
+if [[ "${pid}" ]]
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)